### ISSUE TITLE
Challenge Forum automatically created right after click continue setup up button when create new challenge

### ISSUE THREAD

**Issue by dedywahyudi** [topcoder-platform/work-manager#996]
### Expected behavior
Challenge forum should be created after click Launch as active

### Actual behavior
Challenge Forum automatically created right after click continue setup up button when create new challenge.

### Steps to reproduce the problem
- Go to WM
- Click Launch New
- Fill the form
- Click Continue Set up button
- You'll received email notification about the Challenge forum automatically created
- And you can access from Discussions page: https://discussions.topcoder.com/

### Screenshot/screencast
![Screen Shot 2020-12-16 at 13 56 24](https://user-images.githubusercontent.com/4476442/102315723-3a971400-3fa7-11eb-8810-c2341d7987fc.png)
![Screen Shot 2020-12-16 at 13 58 58](https://user-images.githubusercontent.com/4476442/102315727-3c60d780-3fa7-11eb-8164-698108791b24.png)
![Screen Shot 2020-12-16 at 13 55 21](https://user-images.githubusercontent.com/4476442/102315732-3ec33180-3fa7-11eb-9ef5-895d13585c31.png)

**Comment by acshields** [comment]
@rootelement - thoughts? On the surface, I agree that it makes sense to me that the forum shouldn't be created until the challenge is activated, but there might be something I don't know about.

**Comment by rootelement** [comment]
Nope. This is as designed. When a challenge is in draft, some copilots like to put information into the forums, so it was decided to create the forum right away.

**Comment by dedywahyudi** [comment]
@rootelement I see, but the new challenge forum will be hidden from anyone else right?
